src/qt/createwalletdialog.cpp

```diff
@@ -111,6 +112,28 @@ CreateWalletDialog::~CreateWalletDialog()
     delete ui;
 }
 
+#ifdef ENABLE_EXTERNAL_SIGNER
+void CreateWalletDialog::setSigners(std::vector<ExternalSigner>& signers)
+{
+    if (!signers.empty()) {
+        ui->external_signer_checkbox->setEnabled(true);
+        ui->external_signer_checkbox->setChecked(true);
+        ui->encrypt_wallet_checkbox->setEnabled(false);
+        ui->encrypt_wallet_checkbox->setChecked(false);
+        // The order matters, because connect() is called when toggling a checkbox:
+        ui->blank_wallet_checkbox->setEnabled(false);
+        ui->blank_wallet_checkbox->setChecked(false);
```

Comment:
nit: to be consistent with `gui: create wallet with external signer`, this should be `true` (modulo comment about it being ambiguous anyway).